Description: Respect the OCRODATA environment variable for all lua scripts.
Author: Jakub Wilk <jwilk@debian.org>

Index: ocropus-0.3.1/ocroscript/ocrotoplevel.cc
===================================================================
--- ocropus-0.3.1.orig/ocroscript/ocrotoplevel.cc	2009-11-26 18:47:54.000000000 +0100
+++ ocropus-0.3.1/ocroscript/ocrotoplevel.cc	2009-11-26 18:47:54.000000000 +0100
@@ -471,6 +471,7 @@
     lua_call(L, 0, 0);
 
     // handle OCRODATA environment variable as a directory
+    if(getenv("OCRODATA")) ocroscripts = getenv("OCRODATA");
     lua_pushstring(L, ocrodata);
     lua_setglobal(L, "ocrodata");
 
